upload.js

```diff
@@ -19,18 +19,18 @@ const upload = multer({
 }).single('myImage');
 
 const storages = multer.diskStorage({
-    destination: './public/moreUploads',
-    filename: function (req, files, cb) {
-        cb(null, Date.now() + path.extname(files.originalname));
+    destination: './public/profile',
+    filename: function (req, file, cb) {
+        cb(null, Date.now() + path.extname(file.originalname));
     }
 });
-const uploadfiles = multer({
-    storages: storages,
+const profile = multer({
```

Comment:
Dont upload image to github, In the gitignore 
add 
`public` so that it can ignore all kind of image files.